cmd/csi-sanity/sanity_test.go

```diff
@@ -42,6 +42,7 @@ func init() {
 	flag.StringVar(&config.SecretsFile, prefix+"secrets", "", "CSI secrets file")
 	flag.Int64Var(&config.TestVolumeSize, prefix+"testvolumesize", sanity.DefTestVolumeSize, "Base volume size used for provisioned volumes")
 	flag.StringVar(&config.TestVolumeParametersFile, prefix+"testvolumeparameters", "", "YAML file of volume parameters for provisioned volumes")
+	flag.StringVar(&config.JUnitFile, prefix+"junitfile", "", "JUnit XML file containing test results")
```

Comment:
Minor nit: the description makes and the commit message do not make it clear that this is going to be created as part of the test run.

I suggest "JUnit XML output file where test results will be written".

Reply:
OK, I made the change